datahost-ld-openapi/test/tpximpact/test_helpers.clj

```diff
@@ -12,20 +12,37 @@
   (when-let [db (get system :tpximpact.datahost.ldapi.db/db)]
     (da/destroy db)))
 
-(defn start-system [configs]
-  (-> configs
-      (sys/load-configs)
-      (sys/prep-config)
-      (ig/init)))
+(def ^:dynamic *system* (atom nil))
+
```

Comment:
One of the reasons we stopped using dynamic vars in out larger projects is that creating systems can become increasingly time consuming and expensive as the system grows. We moved away from using `t/use-fixtures :each` for this reason.

Not a big problem right now though.

Additionally, Integrant systems should ideally not be mutable once created.